examples/nodejs/access-control/README.md

```diff
@@ -0,0 +1,40 @@
+<head>
+  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+# Node.js Client SDK
+
+_Read this in other languages_: Coming Soon
+
+<br>
+
+## Example Requirements
+
+- Node version 14 or higher is required
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
```

Comment:
I thought we want to encourage users to use our console to generate a token?

Reply:
Good catch, I copied this from the examples/nodejs/cache/README.md

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/598/commits/d583b64ee0db14557dc380df1c86785c67dbd326